src/components/appShell/AppShell.tsx

```diff
@@ -1,14 +1,17 @@
 import { HomeIcon } from '@camp/design';
 import { AppShell as Shell, MediaQuery } from '@mantine/core';
 import { Outlet, useMatches, useMatchRoute } from '@tanstack/react-location';
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { CreateFamilyButton } from '../CreateFamilyButton';
+import { CreateProjectModal } from '../CreateProject';
 import { CreateProjectButton } from '../CreateProjectButton';
 import { Header, SideBar } from '../organisms';
 import { Breadcrumb } from './Breadcrumb';
 
 export const AppShell = () => {
+  const [isCreateProjectModalOpen, setIsCreateProjectModalOpen] =
```

Comment:
if we're going to use URL change after clicking the create project button, we don't need this state

Reply:
I left it there for our refactoring session